clients/ui/frontend/src/app/pages/modelRegistry/screens/ModelRegistry.tsx

```diff
@@ -7,6 +7,7 @@ import { filterLiveModels } from '~/app/utils';
 import ModelRegistrySelectorNavigator from './ModelRegistrySelectorNavigator';
 import RegisteredModelListView from './RegisteredModels/RegisteredModelListView';
 import { modelRegistryUrl } from './routeUtils';
+import '~/style/MUI-theme.scss';
```

Comment:
I might not understand how all of this works, but are we able to just import this at a higher level so we don't need to import in multiple places?

Reply:
Good point, removed other imports

Reply:
@jenny-s51 you can even add it in `ModelRegistryRoutes.tsx` since it's the entrypoint of MR or even in `App.tsx` since it's the entrypoint of the project if you want.

Reply:
Thank you @lucferbux - moved it to `ModelRegistryRoutes.tsx`